feature-libs/organization/administration/components/shared/item-active.directive.ts

```diff
@@ -19,23 +19,35 @@ export class ItemActiveDirective<T = BaseItem> implements OnInit, OnDestroy {
   ngOnInit() {
     this.subscription = this.itemService.current$
       .pipe(
-        tap((item) => {
-          if (item) {
+        withLatestFrom(this.messageService.get()),
+        tap(([item, messageData]) => {
+          if (item && this.checkDuplicatedMessage(messageData)) {
             this.handleDisabledItems(item);
           }
         })
       )
       .subscribe();
   }
 
+  protected checkDuplicatedMessage(messageData) {
+    return (
+      messageData?.message?.key !== this.message.message.key &&
+      messageData?.type !== this.message.type
+    );
+  }
+
+  protected get message() {
```

Comment:
```suggestion
  export const messageDIsabledItem
```
and analogical for notExist in the other one, then we could move both of them in one file

Reply:
return type `:MessageData`